sky/shell/platform_view.h

```diff
@@ -77,11 +77,14 @@ class PlatformView {
 
   std::unique_ptr<Rasterizer> rasterizer_;
   std::unique_ptr<Engine> engine_;
+  SkyEnginePtr engine_ptr_;
```

Comment:
How about I make the engine maintain a collection of bindings instead of just the single one? That way, you should be able to call Engine::ConnectToEngine multiple times.